src/components/common/NavBar.jsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { Nav, Navbar } from "react-bootstrap";
+import "./index.css";
+
+export const NavBar = () => {
+  return (
+    <div>
+      <Navbar className="navBar">
+        <Navbar.Brand href="/">
+          <img
+            className="brandSpaces"
+            src="https://i.ibb.co/L9GcrMY/LLogo-2.png"
+            alt="Suljak Logo"
+          />
+        </Navbar.Brand>
+
+        <Nav className="collapse navbar-collapse justify-content-end">
+          <Nav.Item className="navSpaces">
+            <Nav.Link className="navLink" href="/Contact">
+              CONTACT US
```

Comment:
please text-transform as we talked about in last PR

Reply:
@Louis This is the old version of NavBar, the new version should be displayed as soon as I solve the conflicts and replace the old version with the new version in many files not only the Navbar
I will push the code with some changes and solve the conflicts then all these issues will disappeared